### PR TITLE
[FIX] accessing self.record fails with multiple records

### LINKED ISSUE #76: [8.0] [Error] account_product_fiscal_classification
When changing a tax on a fiscal classification on a system with multiple `product.template` records sharing that same fiscal classification, [this line of code](https://github.com/OCA/account-fiscal-rule/blob/8.0/account_product_fiscal_classification/models/product_template.py#L86) throws a `ValueError` of "Expected singleton".

### PR DESCRIPTION
**Pull request by thomaspaulb**
Fixes #76

### CODE REVIEW

**Comment by legalsylvain** [comment]
@thomaspaulb could you fix trivial travis error. (flake8) ?

kind regards.

**Comment by thomaspaulb** [comment]
@legalsylvain fixed flake issue

File: account_product_fiscal_classification/models/product_template.py [hunk 1]
```diff
@@ -81,14 +81,16 @@ def write_taxes_setting(self, vals):
         """If Fiscal Classification is defined, set the according taxes
         to the product(s); Otherwise, find the correct Fiscal classification,
         depending of the taxes, or create a new one, if no one are found."""
-        if vals.get('fiscal_classification_id', False):
+        fc_id = vals.get('fiscal_classification_id', False)
+        if fc_id:
             # update or replace 'taxes_id' and 'supplier_taxes_id'
-            classification = self.fiscal_classification_id
+            fc_obj = self.env['account.product.fiscal.classification']
+            fc = fc_obj.browse(classification_id)
```
Comment (legalsylvain) [inline account_product_fiscal_classification/models/product_template.py]: ~classification_id~ -> fc_id

**Review by legalsylvain — CHANGES_REQUESTED** [review]
need fixing.

regards.

**Comment by legalsylvain** [comment]
thanks, fast tracking this patch.

**Pull request merged**